lib/pbench/test/unit/server/test_user_auth.py

```diff
@@ -0,0 +1,365 @@
+import time
+from flask_bcrypt import Bcrypt
+from pbench.server.api.resources.models import UserModel
+
+
+def register_user(
+    client, server_config, email, username, password, firstname, lastname
+):
+    return client.post(
+        f"{server_config.rest_uri}/user",
+        json={
+            "email": email,
+            "password": password,
+            "username": username,
+            "firstName": firstname,
+            "lastName": lastname,
+        },
+    )
+
+
+def login_user(client, server_config, username, password):
+    return client.post(
+        f"{server_config.rest_uri}/session",
+        json={"username": username, "password": password},
+    )
+
+
+class TestUserAuthentication:
+    @staticmethod
```

Comment:
I'm curious why people choose to use static test methods instead of instance methods? Is there a particular reason you prefer them? I always figure `self, ` in the test signature is less typing. :-)

Reply:
Not sure but I think its just because these methods dont need any class/self attributes internally and even tho they belong to the same class they just act independent of each other and methods does not depend on the state of the object itself.